test/SwooleRequestHandlerRunnerTest.php

```diff
@@ -104,7 +112,14 @@ public function testRunRegistersExpectedHttpServerListenersAndStartsServerWhenIn
 
     public function testRunRegistersExpectedHttpServerListenersAndStartsServerWhenInProcessMode(): void
     {
-        $this->httpServer->mode = SWOOLE_PROCESS;
+        if (PHP_VERSION_ID >= 80200 && PHP_VERSION_ID < 80300) {
+            $this->markTestSkipped('Test relies on modifying readonly properties, which is not allowed in PHP 8.2+');
+        }
+
+        $reflection = new ReflectionProperty($this->httpServer, 'mode');
+        /** @psalm-suppress UnusedMethodCall */
+        $reflection->setAccessible(true);
```

Comment:
This is unnecessary since 8.1
https://www.php.net/manual/en/reflectionproperty.setaccessible.php